keanu-project/src/test/java/io/improbable/keanu/vertices/dbl/DoubleVertexSamplesTest.java

```diff
@@ -58,4 +56,20 @@ public void doesCalculateVariance() {
         );
         assertThat(variances, allCloseTo(1e-8, expectedValues));
     }
+
+    @Test
+    public void canGetSamplesAsTensor() {
+        List<DoubleTensor> samplesAsList = samples.asList();
+
+        DoubleTensor samplesAsTensor = samples.asTensor();
+        List<DoubleTensor> samplesAsTensorSliced = samplesAsTensor.sliceAlongDimension(0, 0, samplesAsTensor.getShape()[0]);
```

Comment:
You can get this list by doing         
`List<DoubleTensor> samplesAsTensorSliced = samplesAsTensor.split(0,1,2,3,4);`